src/main/java/Duke.java

```diff
@@ -1,10 +1,294 @@
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static void main(String[] args) throws DukeException {
+        // String home = System.getProperty("user.home");
+        // Path path = Paths.get(home, "OneDrive","Documents", "NUS",
+        //         "Y2S2", "CS2103", "Individual Project", "duke.txt");
+        String relativePath = "data/duke.txt";
```

Comment:
This can perhaps be a constant with the name "RELATIVE_PATH" since it will remain unchanged throughout.

Reply:
thankss, updated